### PR TITLE
support prereq check for MCE v2.0

### PR DESCRIPTION
**Pull request by chrisahl**
For issue https://issues.redhat.com/browse/CMCS-51

### CODE REVIEW

File: controllers/installer/installer_controller.go [hunk 1]
```diff
@@ -521,18 +521,32 @@ func (r *IDPConfigReconciler) getRHACMCRList() (*unstructured.UnstructuredList,
...
+		cms_v1alpha1, err_v1alpha1 := r.getMCEConfigMapList("v1alpha1")
+		if (err_v1alpha1 == nil) && (len(cms_v1alpha1.Items) > 0) {
+			found = true
+		}
 	}
-	if len(cms.Items) == 0 {
+
+	if found {
+		return true, nil
+	} else {
```
Comment (itdove) [inline controllers/installer/installer_controller.go:542]: the "else" is not needed, but it is a detail

**Comment by chrisahl** [comment]
quay image with the fix is 0.0.1-20220325-17-36-50-0749933-main

**Pull request merged**